### DISCUSSION TITLE
what is react-native link replaced with?

### DISCUSSION THREAD

**Discussion by aep** [react-native-webview/react-native-webview#3592] [Q&A] [answered]
Hi,

super new to react native and this is the first thing i'm trying to do on macos.
i got `RNCWebViewModule could not be found` and the readme clearly says i need to run react-native link.

However, that appears already deprecated
https://stackoverflow.com/a/72784327

according to this: https://reactnative.dev/docs/linking-libraries-ios
its supposed to magically just work

i also tried doing the manual step there, by dragging the xcode thing into the other xcode thing.
but the .a from webview is never built so it fails with 

```
clang++: error: no such file or directory: '/Users/aep/Library/Developer/Xcode/DerivedData/poop-cznejfbgnvhhmncjarhpbjbsjzan/Build/Products/Debug-iphoneos/libRNCWebView.a'
```

i also cant just build it because the xcodeproj from react-native-webview doesnt build

```
error: Build input file cannot be found: '/Users/aep/kram/poop/node_modules/react-native-webview/apple/RNCWebViewManager.m'. Did you forget to declare this file as an output of a script phase or custom build rule which produces it? (in target 'RNCWebView-macOS' from project 'RNCWebView')
```

**Comment by DanielSRS** [comment, marked as answer]
> the readme clearly says i need to run react-native link.

The readme says to run `react-native link` if your react native version is lower than 0.60 (released 5 years ago). Current react native version is 0.76

You only need to run `pod install` inside the `ios` or `macos` folder

**Reply by aep** [reply]
Thanks. This didn't work for me. I gave up. But maybe it helps someone else.
